src/Workspaces/CoreTest/Differencing/LongestCommonSubsequenceTests.cs

```diff
@@ -0,0 +1,311 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using Xunit;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Differencing.UnitTests
+{
+    public class LongestCommonSubsequenceTests
+    {
+        LongestCommonSubsequenceString lcs = new LongestCommonSubsequenceString();
+
+        private class LongestCommonSubsequenceString : LongestCommonSubsequence<string>
```

Comment:
>LongestCommonSubsequenceString [](start = 22, length = 30)

We already have LongestCommonSubstring class. 
Do we need this one?

[http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Differencing/LongestCommonSubstring.cs,55843be7ccb0c072,references](http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Differencing/LongestCommonSubstring.cs,55843be7ccb0c072,references)